lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -1104,7 +1141,7 @@ sub process_and_run_upgrade_script {
         template => $template,
         no_auto_output => 1,
         format_options => {extension => 'sql'},
-        output_file => 'upgrade',
+        output_file => 'upgrade.sql',
```

Comment:
If we are specifying an extension it should be recognised and managed by the template engine.  Taking a quick look